test/graph_rewrites/backend_manager_test.cc

```diff
@@ -40,21 +40,23 @@ namespace testing {
 
 /*
 These tests test the Backend Handling by the bridge
-Since the backend is set globaly the setting of NGRAPH_TF_BACKEND might cause
+Since the backend is set globally the setting of NGRAPH_TF_BACKEND might cause
 some tests to fail
-For e.g. "BackendCLustering" test would fail If the NGRAPH_TF_BACKEND is set to
+For e.g. "BackendCLustering" test would fail if the NGRAPH_TF_BACKEND is set to
 INTERPRETER the nodes going to CPU also go to interpreter
 putting all the nodes in the same cluster.
 */
 
 // Test SetBackendAPI
 TEST(BackendManager, SetBackend) {
   ASSERT_OK(BackendManager::SetBackendName("CPU"));
-  string cpu_backend = BackendManager::GetCurrentlySetBackendName();
+  string cpu_backend;
```

Comment:
We would want to disable these tests when BackendManager::GetCurrentlySetBackendName() is not CPU